DuckDuckGo/NetworkProtectionDebugViewController.swift

```diff
@@ -95,19 +104,11 @@ final class NetworkProtectionDebugViewController: UITableViewController {
                 break
             }
 
+        case .debugFeature:
+            configure(cell, forDebugFeatureAtRow: indexPath.row)
+
         case .simulateFailure:
-            switch SimulateFailureRows(rawValue: indexPath.row) {
-            case .controllerFailure:
-                cell.textLabel?.text = "Enable NetP > Controller Failure"
-            case .tunnelFailure:
-                cell.textLabel?.text = "Enable NetP > Tunnel Failure"
-            case .crashFatalError:
-                cell.textLabel?.text = "Tunnel: Crash (Fatal Error)"
-            case .crashMemory:
-                cell.textLabel?.text = "Tunnel: Crash (CPU/Memory)"
-            case .none:
-                break
-            }
+            configure(cell, forSimulateFailureAtRow: indexPath.row)
```

Comment:
I didn't want to do this change, but I was getting a cyclomatic complexity alert in this method, due to my additions.

I could've added a flag to disable the warning but I figured we'll probably prefer to do things right.

Reply:
This change is causing release builds to fail. You’ll need to either move the `configure` function above the `NETWORK_PROTECTION` compiler flag. Sorry, this is a bit fiddly. I need to address this properly as the options shouldn’t be visible at all in release builds…

Reply:
Thank you, fixed it for release builds so that they show an empty VC for now.